cypress-io/cypress
# fix: add top-level run-all button to allow all specs to be run at once

- Closes #24759 

### User facing changelog
experimentalRunAllSpecs: Allow running all specs regardless of folder structure by adding a top-level run-all button.

### Additional details
We only allowed running "all specs" if the user happened to have one root folder e.g. storing all specs under `cypress/e2e`. If they had two root folders containing specs, this wasn't possible.

There is now a "top-level" run all button in both the Specs Page SpecsList and the Runner SpecsList.

DRAFT: Need to verify InlineSpecList button with design.

### Steps to test
Checkout this branch, choose a project and add `experimentalRunAllSpecs` to the `e2e` config. Run `yarn watch` and `yarn cypress:open` in two separate terminals and then choose your project. Verify the Specs page and the Runner has the top-level "Run All Specs" button and that it functions correctly.

### How has the user experience changed?

https://user-images.githubusercontent.com/25158820/204351144-6a502ef1-8d24-4b1f-9864-56eb2665f34e.mov



### PR Tasks
